Android/app/src/main/java/fashiome/android/models/Product.java

```diff
@@ -136,47 +112,87 @@ public String getCurrency() {
         return this.currency;
     }
 
-    public void setCurrency(String currency) {
-        this.currency = currency;
-        put("currency", currency);
+    public String getProductName() {
+
+        if (this.productName == null) {
+            this.productName = getString("productName");
+        }
+        String string= this.productName;
+        return string;
     }
 
-    public String getProductDescription() {
+    public String getProductSize() {
 
-        if (this.productDescription == null) {
-            this.productDescription = getString("productDescription");
+        if (this.productSize == null) {
+            this.productSize = getString("productSize");
         }
-        String string= this.productDescription;
+        String string = this.productSize;
         return string;
     }
 
+    public List<String> getPhotos() {
```

Comment:
This method didn't fetch from parse if the photos were null so it's important to add the parse call from here


Thanks,
Rakhee

On Mar 19, 2016, at 2:59 PM, Deepak Barge <notifications@github.com<mailto:notifications@github.com>> wrote:


In Android/app/src/main/java/fashiome/android/models/Product.java<https://github.com/deborshisaha/ModaRenta/pull/57#discussion_r56755764>:

>          return string;
>      }
>
> +    public List<String> getPhotos() {


you can get the photos from the cloudinary method. any specific need to add this ?

-
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub<https://github.com/deborshisaha/ModaRenta/pull/57/files/b5a0e76ed0e25c34f4b9737bf76df95ed8676ffb#r56755764>